src/Paprika/Store/RootPage.cs

```diff
@@ -240,14 +240,14 @@ public void Destroy(IBatchContext batch, in NibblePath account)
     }
 }
 
-[StructLayout(LayoutKind.Explicit, Size = Size, Pack = 1)]
+[StructLayout(LayoutKind.Sequential, Pack = sizeof(byte), Size = Size)]
```

Comment:
Persistent

Reply:
Sequential is fine, just don't move fields 🙂